src/config.toml

```diff
@@ -8,9 +8,9 @@ baseurl = "https://dex.decred.org/"
   description = "DCRDEX is an open source, peer-to-peer cryptocurrency exchange built by the Decred Project."
 
   # Download URLs
-  win_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.6/decrediton-v1.7.6.exe"
-  mac_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.6/decrediton-amd64-v1.7.6.dmg"
-  m1_decrediton_url  = "https://github.com/decred/decred-binaries/releases/download/v1.7.6/decrediton-arm64-v1.7.6.dmg"
-  lin_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.6/decrediton-v1.7.6.AppImage"
-  tar_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.6/decrediton-v1.7.6.tar.gz"
-  dexreleases_url     = "https://github.com/decred/dcrdex/releases/latest"
+  win_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-v1.7.7.exe"
+  mac_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-amd64-v1.7.7.dmg"
+  m1_decrediton_url  = "https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-arm64-v1.7.7.dmg"
+  lin_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-v1.7.7.AppImage"
+  tar_decrediton_url = "https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-v1.7.7.tar.gz"
```

Comment:
The latest lacks a "v" in the decred-binaries tag.  @jrick Is that intentional and should we update this, or are you guys going to re-tag?

Reply:
it's been fixed